Automattic/portkey-android
# Upgrade to gradle 6.1.1

This PR updates gradle wrapper to `6.1.1`. The wrapper file changes are created automatically by the `gradle wrapper --gradle-version 6.1.1` command.

[As of Android Gradle Plugin 3.1 `buildToolsVersion` is not necessary anymore as the plugin uses the minimum required version by default.](https://developer.android.com/studio/releases/gradle-plugin#3-1-0) If this value is off, it creates warnings such as the one below, so it's best to just remove it. 

```
WARNING: The specified Android SDK Build Tools version (29.0.1) is ignored, as it is below the minimum supported version (29.0.2) for Android Gradle Plugin 4.0.1.
Android SDK Build Tools 29.0.2 will be used.
To suppress this warning, remove "buildToolsVersion '29.0.1'" from your build.gradle file, as each version of the Android Gradle Plugin now has a default version of the build tools.
```

**To test:**

Running `./gradlew build` and verifying the build is successful should be enough.

---

I don't think this will matter, but it might be best to approve this PR but not merge it until we are ready to merge the [WPAndroid PR](https://github.com/wordpress-mobile/WordPress-Android/pull/12896) which is close to be ready for review. I'll leave the decision for that to you @aforcier & @mzorz!
